recipes/phytop/meta.yaml

```diff
@@ -0,0 +1,159 @@
+package:
+  name: phytop
+  version: 1.0
+  
+source:
+  #git_url: https://github.com/zhangrengang/phytop.git
+  path: /media/nfs3/qxc2/biosoftware/phytop_condabuild/phytop
```

Comment:
What is this ?
You should get the sources from some repository, not commit them here.